server/database/seeds/20190729181912-user.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
Why this?

Reply:
same reason as above which is `I created it to test the login route with a user seeded into the database`